de.dlr.sc.virsat.server.test/src/de/dlr/sc/virsat/server/dataaccess/TransactionalJsonProviderTest.java

```diff
@@ -118,16 +123,52 @@ public void testWriteTo() throws WebApplicationException, IOException {
 		writeToAndAssert();
 	}
 	
+	/**
+	 * Open the remote repository and count the commits
+	 * @return number of commits in remote
+	 * @throws IOException
+	 * @throws NoHeadException
+	 * @throws GitAPIException
+	 * @throws InterruptedException
+	 */
+	private int countCommits() throws IOException, NoHeadException, GitAPIException, InterruptedException {
+		Git git = Git.open(pathRepoRemote.toFile());
+		Iterator<RevCommit> iterator = git.log().call().iterator();
+		int commits = 0;
+		while (iterator.hasNext()) {
+			System.out.println(iterator.next().getFullMessage());
```

Comment:
Do we want to print out the messages here?

Reply:
Nope, I will delete the print statement